packages/material-ui/src/Avatar/Avatar.js

```diff
@@ -1,60 +1,24 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import styled from 'styled-components';
+import themed from '../styles/themed';
 import Person from '../internal/svg-icons/Person';
 
-export const styles = (theme) => ({
+export const styles = {
```

Comment:
For future reference: Should be possible to move this into the typescript definitions and get rid of the size cost + get documentation on the classes in the editor. Would need to update the documentation generation to read from the proptypes instead though.